base/applications/sndrec32/sndrec32.cpp

```diff
@@ -880,6 +922,10 @@ WndProc(HWND hWnd,
             break;
 
         case WM_DESTROY:
+            if(hfButtoncaption)
+            {
+                DeleteObject(hfButtoncaption);
+            }
```

Comment:
```suggestion
            if (hfButtonCaption)
                DeleteObject(hfButtonCaption);
```

Reply:
Do we need to keep that `hfButtoncaption` around for the whole duration of the program?

Reply:
Not sure. The documentation states:

> When you no longer need the font, call the [DeleteObject](https://docs.microsoft.com/en-us/windows/desktop/api/wingdi/nf-wingdi-deleteobject) function to delete it.

I thought that this seems to be the correct place. When you as an experienced Win32 Dev tell me something else I'd like glad to hear your opinion. :)

Reply:
> When you no longer need the font

Exactly, nowhere else you use that font handle, except in your helper function where you create it to do some stuff just afterwards. Thus the handle can be deleted at the end of said helper function.

Reply:
OK. I tried to reduce the usage of this HFONT by making it a variable in the scope of WndProc() instead of a global one but deleting the Object direct after using it breaks font drawing.